WebDriverAgentLib/Categories/XCUIElement+FBUID.m

```diff
@@ -43,6 +49,20 @@ + (void)load
   NSAssert(XCElementSnapshotCls != nil, @"Could not locate XCElementSnapshot class");
   Method uidMethod = class_getInstanceMethod(self.class, @selector(fb_uid));
   class_addMethod(XCElementSnapshotCls, @selector(fb_uid), method_getImplementation(uidMethod), method_getTypeEncoding(uidMethod));
+  
+  // Support for Xcode 14.3 requires disabling the new predicate validator, see https://github.com/appium/appium/issues/18444
+  Class XCTElementQueryTransformerPredicateValidatorCls = objc_lookUpClass("XCTElementQueryTransformerPredicateValidator");
+  if (XCTElementQueryTransformerPredicateValidatorCls != nil)
+  {
+    Method validatePredicateMethod = class_getClassMethod(XCTElementQueryTransformerPredicateValidatorCls, NSSelectorFromString(@"validatePredicate:withExpressionsAllowed:"));
+    if (nil != validatePredicateMethod) {
```

Comment:
IMHO positive if conditions are easier to read than negative ones. Would it make sense to "flip" this one?